docs/technical-documentation/testing-notes.md

```diff
@@ -1,43 +1,12 @@
 ## How to find things in Fedora?
 
-The map between Drupal URIs and their corresponding Fedora URIs is stored in a service called Gemini. An Islandora "pseudo-field" can be enabled that will display the corresponding Fedora URI on the page of a node, taxonomy term, or media object.  You can also query Gemini directly. Instructions for doing that are provided below.
+For any Drupal URI, the corresponding Fedora URIs is computed in a service called Milliner. In Drupal, a "pseudo-field" is available that will display the corresponding Fedora URI on the page of a node, taxonomy term, or media object. The presence of this URI does NOT guarantee the existence of the corresponding entity in Fedora. 
 
-For more information about the Gemini service, see the [Gemini README](https://github.com/Islandora/Crayfish/tree/main/Gemini).
+### Enabling the Fedora URI Pseudo-field
 
-## Fedora URI Pseudo-field
+To display the Fedora URI pseudo-field on a Drupal node, media, or taxonomy term, go to Manage > Configuration > Islandora and select all the bundles for which you would like the Fedora URI displayed. Once you have selected the bundles, and cleared the cache, the new pseudo-field will appear at the bottom of _all display modes_. You can alter where in the display the Fedora URI field appears, by going to the "Manage Display" page for the bundle. For example, for a Repository Item, you'd go to http://localhost:8000/admin/structure/types/manage/islandora_object/display. In that list you should see `Fedora URI` which you can move around (or hide) as desired. Clearing cache may be necessary to refresh the node display.
```

Comment:
Manage > Configuration > Islandora > Core Settings and in the section "Fedora URL Display"

Also, for consistency I'd use the same 'breadcrumb' navigation style for directing users to the Manage Display page of the example: Manage > Structure > Content Types > Select "Manage Display" from drop down menu for Repository Item.

Reply:
Thanks! Done.